public/assets/js/confirm-modal.js

```diff
@@ -0,0 +1,12 @@
+document.addEventListener("DOMContentLoaded",(
```

Comment:
Mss goed om maar gelijk met Webpack encore.  Typescript e.d. aan de slag te gaan onder de noemer "uitstel is afstel"?

Reply:
Prima, laten we in de volgende PR hier samen even naar kijken

Reply:
Check